AUTHORS.txt

```diff
@@ -15,3 +15,4 @@ Joseph Mittelstaedt
 Troy Fox
 Vikram Sekar
 Casper Schippers
+Sumatran Tiger
```

Comment:
Please also add yourself to `docs/about/authors.rst`.

Reply:
Do I need add myself to AUTHORS.txt ? Will contributor be added to AUTHORS.txt  by owner when related version release?